fe1-web/src/features/lao/hooks/LaoHooks.ts

```diff
@@ -0,0 +1,78 @@
+import FeatureContext from 'core/contexts/FeatureContext';
+import { useContext } from 'react';
+import { useSelector } from 'react-redux';
+import { LaoReactContext, LAO_FEATURE_IDENTIFIER } from '../interface';
+import { Lao } from '../objects';
+import {
+  selectCurrentLao,
+  selectCurrentLaoId,
+  selectIsLaoOrganizer,
+  selectLaosList,
+  selectLaosMap,
+} from '../reducer';
+
+export namespace LaoHooks {
```

Comment:
Just to make sure I understand your thinking, why do we need a namespace here?

Reply:
That is one of the things I am not sure about. We could move the hooks below `/** Hooks defined by the lao feature */` outside the namespace without any problem :)

The question is if we would like to have the "real" `useLaoList` outside a namespace or if it should be part of the `LaoHooks` namespace. What do you think? (If e.g. feature x imports it, it will be in `XHooks` to avoid name clashes)

Reply:
In principle we could achieve that same result without having a namespace here, but then doing the following in the `hooks/index.ts`
```
// ES 2020, not sure if supported in our project
export * as LaoHooks from './LaoHooks';
// or
import * as LaoHooks from './LaoHooks';
export LaoHooks;
```

Reply:
If somewhere else there's a conflict between 2 imports, we could also easily do something ugly like
`import { useLaoList as LaoHooks_useLaoList } from ...`
or you know, using a less-ugly-name 😅

Reply:
The auto completion of the IDE won't be as nice anymore but we can do that too, yes. What do you prefer?